Mathlib/Data/Matroid/Circuit.lean

```diff
@@ -319,4 +336,128 @@ lemma ext_iff_circuit {M₁ M₂ : Matroid α} :
     M₁ = M₂ ↔ M₁.E = M₂.E ∧ ∀ C, M₁.Circuit C ↔ M₂.Circuit C :=
   ⟨fun h ↦ by simp [h], fun h ↦ ext_circuit h.1 fun C hC ↦ h.2 (C := C)⟩
 
+section Elimination
+
+/-! ### Circuit Elimination -/
+
+variable {ι : Type*} (x : ι → α) {z : α} {C₁ C₂ : Set α}
+
+/-- A version of `Matroid.Circuit.strong_multi_elimination` that is phrased using insertion. -/
+lemma strong_multi_elimination_insert (I : ι → Set α) (hxI : ∀ i, x i ∉ I i) {J : Set α}
```

Comment:
```suggestion
lemma Circuit.strong_multi_elimination_insert (I : ι → Set α) (hxI : ∀ i, x i ∉ I i) {J : Set α}
```
for consistency with the next one

Reply:
I agree with @apnelson1 here.

Reply:
I do still think it's a good idea. I agree that the value of dot notation here is diminished, but the benefits are still there:
- the family of elimination lemmas all have similar names, which aids understanding when reading the docs
- when searching or using vscode completion, the substring "Circuit" still shows up

Reply:
I am on the side of having a better dot notation.

Reply:
I've added it into the namespace. The dot notation isn't worse, it is just unavailable for this lemma in either case.